src/config/Config.ts

```diff
@@ -178,6 +178,7 @@ const DEFAULT_PROPERTIES: Properties = {
     'hazelcast.client.heartbeat.interval': 5000,
     'hazelcast.client.heartbeat.timeout': 60000,
     'hazelcast.client.invocation.retry.pause.millis': 1000,
+    'hazelcast.client.schema.max.put.retry.count': 100,
```

Comment:
Can you update ConfigBuilder.validateProperty and ConfigBuilderTest as well? We should add validation and test for validation. 

@mdumandag Should we document this property? Is it public?

Reply:
Fixed